packages/customWidgets/image-viewer-web/src/package.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <package xmlns="http://www.mendix.com/package/1.0/">
-    <clientModule name="ImageViewer" version="1.3.3" xmlns="http://www.mendix.com/clientModule/1.0/">
+    <clientModule name="ImageViewer" version="1.3.2" xmlns="http://www.mendix.com/clientModule/1.0/">
```

Comment:
Are we supposed to go down in version? 😄

Reply:
Yup